hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -544,6 +545,15 @@ public String getBucketKeyPrefixFSO(String volume, String bucket) throws IOExcep
     return getOzoneKeyFSO(volume, bucket, OM_KEY_PREFIX);
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public Pair<Long, Long> getVolumeBucketIdPairFSO(String fsoKey) {
+    String[] keySplit = fsoKey.split(OM_KEY_PREFIX);
+    return Pair.of(Long.parseLong(keySplit[1]), Long.parseLong(keySplit[2]));
```

Comment:
Please handle exceptions related to array size and number parsing.  Isn't there some method for parsing FSO keys already?

Reply:
No this is the only place. However I am throwing a checked exception if there is an exception while parsing